clang/include/clang/Basic/DiagnosticOptions.h

```diff
@@ -108,6 +108,9 @@ class DiagnosticOptions : public RefCountedBase<DiagnosticOptions>{
   /// The file to serialize diagnostics to (non-appending).
   std::string DiagnosticSerializationFile;
 
+  /// File that defines suppression mappings.
```

Comment:
NIT: diagnostic suppression mappings?

I can imagine being confused had I not had this context ("what are supressions and why we need to map them?")

Reply:
I was dropping that mostly because this is in `DiagnosticOptions`, but adding those to names to be consistent with others here.